src/main/resources/migration/common/V10__create_test_data.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE test_data
```

Comment:
instead of having a test_data table why not have one of the real; tables with minimal columns; e.g. `licence`

Reply:
I just wanted a full, end-to-end slice of API that I could leave in as an example for now. Was planning to create the licence table as one of the next pieces of work.